package.json

```diff
@@ -69,7 +69,8 @@
     "css-vars-ponyfill": "^2.3.2",
     "date-fns": "^2.16.1",
     "dompurify": "^3.0.1",
-    "prop-types": "^15.7.2"
+    "prop-types": "^15.7.2",
+    "ts-pattern": "^4.2.2"
```

Comment:
Also comment on rollup.config.externals: `we dont include ts-pattern to avoid dependency`

Reply:
I did it in here https://github.com/sendbird/sendbird-uikit-react/pull/487/commits/63a417b2a8b0ea43b4d8824d57c914a135b104b4

Reply:
I move it to as you said https://github.com/sendbird/sendbird-uikit-react/pull/487/commits/63a417b2a8b0ea43b4d8824d57c914a135b104b4. But another problem came up. 
It will